### PR TITLE
Add e2e-architecture to README.md

### PR DESCRIPTION
**Pull request by MMMsc**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,5 @@ test/setup/config/
 
 .tmp
 .idea
+
+.DS_Store
```
Comment (clyang82) [inline .gitignore:33]: Do not suggest to add this temporary file?

File: README.md [hunk 2]
```diff
@@ -92,6 +92,9 @@ _Note:_ This will delete Multicluster Global Hub Operator and the CRD from the c
 make undeploy
 ```
 
+## E2E-Tests
+![e2eArchitecture](doc/architecture/e2e-global-hub-arch.png)
+
```
Comment (yanmxa) [inline README.md:97]: suggestion
- E2E-Tests -> E2E Tests
- e2eArchitecture -> E2E Architecture
- e2e-global-hub-arch -> multicluster-global-hub-e2e-arch.png

**Pull request merged**